browser/brave_ads/brave_ads_host.cc

```diff
@@ -0,0 +1,120 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_ads/brave_ads_host.h"
+
+#include <memory>
+#include <string>
+#include <utility>
+
+#include "brave/browser/brave_ads/ads_service_factory.h"
```

Comment:
this entire file needs to be in an ENABLE_EXTENSIONS guard see https://github.com/brave/brave-core/pull/9611/files#r716881243